website/redirects.next.js

```diff
@@ -84,11 +104,6 @@ module.exports = [
     destination: '/docs/commands/:path*',
     permanent: true,
   },
-  {
-    source: '/docs/extend/:path*',
```

Comment:
when should we use `:path*` instead of  `:path`? 🤔

Reply:
Good question `:path` will match for a single path (e.g packer.io/docs/extend/plugins) where `:path*` will match multiple paths (e.g. packer.io/docs/extend/plugins and packer.io/docs/extend/plugins/..../....). 

I planned to come back and leave a comment here, but you you are too fast for me. I removed this redirect because I believe it will cause issues since the new extending packer paths are nested (e.g packer.io/docs/plugins/creation/custom-builders). Before they were all on the same level `packer.io/docs/extending/<path>` so the redirect made sense.

Reply:
Ahh ok, `:path*` is recursive!